projects/libvips/project.yaml

```
@@ -5,9 +5,20 @@ auto_ccs:
   - "kleisauke@gmail.com"
   - "lovell.fuller@gmail.com"
 main_repo: 'https://github.com/libvips/libvips'
-
 fuzzing_engines:
   - afl
   - honggfuzz
   - libfuzzer
-
+coverage_extra_args: \
+  -ignore-filename-regex=*/src/aom/* \
+  -ignore-filename-regex=*/src/cgif/* \
+  -ignore-filename-regex=*/src/highway/* \
+  -ignore-filename-regex=*/src/lcms/* \
+  -ignore-filename-regex=*/src/libexif/* \
+  -ignore-filename-regex=*/src/libheif/* \
+  -ignore-filename-regex=*/src/libimagequant/* \
+  -ignore-filename-regex=*/src/libjpeg-turbo/* \
+  -ignore-filename-regex=*/src/libspng/* \
+  -ignore-filename-regex=*/src/libtiff/* \
+  -ignore-filename-regex=*/src/libwebp/* \
+  -ignore-filename-regex=*/src/zlib/*
```

It looks like we could just pass `-sources=src/libvips` as llvm-cov argument, this way it will only generate coverage for files under those directories.

```suggestion
coverage_extra_args: -sources=src/libvips
```

I verified whether this worked with:
```console
$ python infra/helper.py build_image libvips
$ python infra/helper.py build_fuzzers --sanitizer=coverage libvips
$ python infra/helper.py coverage libvips -- -sources=src/libvips
```

---

Ah, the initial suggestion was `-sources=src/libvips`, which doesn't work, it should be `-sources src/libvips`.

---

Ah, nice!